### PR TITLE
Print alerts to stderr when using the CSV writer

### PR DESCRIPTION
**Pull request by nickelization**
Also print "invalid format" errors to stderr.

### CODE REVIEW

File: src/clique_csv_writer.erl [hunk 1]
```diff
@@ -32,8 +32,14 @@
 
 -spec write(status()) -> iolist().
```
Comment (macintux) [inline src/clique_csv_writer.erl]: spec needs to be fixed

**Comment by macintux** [comment]
Once the dialyzer fix is in, I think we're good.

**Comment by nickelization** [comment]
Thanks @macintux, not sure how I missed that before - just rebased and pushed a fix.

**Comment by nickelization** [comment]
@borshop merge

**Pull request merged**